shell/platform/android/io/flutter/plugin/platform/VirtualDisplayController.java

```diff
@@ -199,6 +210,16 @@ public void dispose() {
     renderTarget.release();
   }
 
+  @TargetApi(31)
+  private void resize31(
+      View embeddedView, int width, int height, final Runnable onNewSizeFrameAvailable) {
+    renderTarget.resize(width, height);
+    // On Android versions 31+ resizing of a Virtual Display's Presentation is natively supported.
+    // Fix in: https://github.com/flutter/flutter/issues/128920
```

Comment:
Please remove the comment "// Fix in: ..." as it isn't needed in the code.

Reply:
OK, this line has removed. I have merged `main` into `pre/128920`.